addons/mail/models/mail_activity_mixin.py

```diff
@@ -182,7 +182,7 @@ def _search_activity_state(self, operator, value):
                     ON res_users.id = mail_activity.user_id
              LEFT JOIN res_partner
                     ON res_partner.id = res_users.partner_id
-                 WHERE mail_activity.res_model = %(res_model_table)s
+                 WHERE mail_activity.res_model = %(res_model_table)s AND mail_activity.active = true 
```

Comment:
Hey @pyduf

I thought about it but I see no issue here, as we only restrict the result of the search / read_group to only consider active activities. I see there is an option to keep done activities, per activity type. Do we delete the activity if not enabled, and archive it otherwise? Thanks for the info

Not much to say on the test either, LGTM :)